### PR TITLE
updates to web ui to be able to be used behind reverse proxy, added a…

### PR DESCRIPTION
**Pull request by mswiderski**
…pplication info loading, extended showError to include details and error ids

### CODE REVIEW

File: tools/webui/src/main/resources/META-INF/resources/webjars/timefold/js/timefold-webui.js [hunk 1]
```diff
@@ -26,30 +26,69 @@ function replaceTimefoldAutoHeaderFooter() {
...
   notification.toast({delay: 30000});
   notification.toast('show');
 }
 
+// ****************************************************************************
+// Application info
+// ****************************************************************************
+
+function applicationInfo() {
+  $.getJSON("tf/info", function (info) {
```
Comment (rsynek) [inline tools/webui/src/main/resources/META-INF/resources/webjars/timefold/js/timefold-webui.js:85]: Maybe remove the prefix, see https://github.com/TimefoldAI/timefold-platform/pull/61#discussion_r1294405188

**Review by rsynek — APPROVED** [review]
One suggestion related to the platform PR. 

Also, please make sure the commit message complies with https://www.conventionalcommits.org/en/v1.0.0/.

File: tools/webui/src/main/resources/META-INF/resources/webjars/timefold/js/timefold-webui.js (hunk 1 above)
Comment (mswiderski) [reply tools/webui/src/main/resources/META-INF/resources/webjars/timefold/js/timefold-webui.js]: moved it to /info

**Comment by mswiderski** [comment]
commit message adjusted, does it look good now?

**Pull request merged**